python/composio/tools/toolset.py

```diff
@@ -161,6 +163,10 @@ def __init_subclass__(
             )
         cls._description_char_limit = description_char_limit or 1024
         cls._action_name_char_limit = action_name_char_limit
```

Comment:
Consider adding validation for `action_name_char_limit` similar to `description_char_limit`. Currently, there's no warning or validation if it's None, and no enforcement of positive integer values. This could lead to unexpected behavior if invalid values are provided.